src/shared/components/case-list/case-list.component.ts

```diff
@@ -14,7 +15,8 @@ export class DateTimeFormatUtils {
 @Component({
   selector: 'ccd-case-list',
   templateUrl: './case-list.component.html',
-  styleUrls: ['./case-list.component.scss']
+  styleUrls: ['./case-list.component.scss'],
+  providers: [BrowserService]
```

Comment:
can this providers move to case-list.module.ts?

Reply:
done